### PR TITLE
Added about Final keyword in java.md

### PR DESCRIPTION
**Pull request by kaushal-mohit**
explained about final variable, final method, final class in java

### CODE REVIEW

File: java.md [hunk 1]
```diff
@@ -132,6 +132,73 @@ Class Sum {
...
+### 2. Final Method
+If any method is declared with keyword final, then it can't be overridden.
+
+### Example
+
+```java
+class Fruit{
+    //final method
+    final void rate(){
+        System.out.println("Rate is Rs. 100");
```
Comment (onecompiler-ops) [inline java.md:167]: Add globally acceptable examples

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]

**Comment by kaushal-mohit** [comment]
Hey! @onecompiler-ops  , please explain me what are globally acceptable examples? Because these examples are absolutely right.

**Comment by onecompiler-ops** [comment]
@kaushal-mohit How does some one referring from USA/ Germany understand `Rs.` ?

**Comment by onecompiler-ops** [comment]
Fix code formatting as well

**Review by kaushal-mohit — COMMENTED** [review]
Changes done.

**Pull request closed without merging**